### PR TITLE
Migrate gci-m59 jobs to be handled by boskos

### PR DESCRIPTION
**Pull request by krzyzacy**
ref https://github.com/kubernetes/test-infra/pull/4481

Convince @wonderfly this is going to work, before flip all gci-qa jobs.

/assign @wonderfly

### CODE REVIEW

**Comment by wonderfly** [comment]
cc/ @kubernetes/goog-image 

/lgtm Will we have access (at least Editor) to the shared projects?

**Comment by krzyzacy** [comment]
hummmm, m59-slow is fine, the other two are having trouble up the cluster -- @wonderfly any clue?

**Comment by wonderfly** [comment]
Let me take a look

File: jobs/config.json [hunk 1]
```diff
@@ -2366,10 +2364,10 @@
     "args": [
       "--check-leaked-resources",
       "--env-file=jobs/platform/gce.env",
-      "--env-file=jobs/env/ci-kubernetes-e2e-gce-gci-qa-slow-m59.env",
       "--extract=gci/gci-59",
       "--gcp-project=e2e-gce-gci-qa-slow-m59",
```
Comment (krzyzacy) [inline jobs/config.json]: grrr, forgot to remove this line -- (I should have a presubmit guarding, but)

**Pull request merged**